### DISCUSSION TITLE
Questions about explicit waters, choice of force field

### DISCUSSION THREAD

**Discussion by rwxayheee** [Electrostatics/apbs#179] [Q&A]
Dear APBS experts, 

I’m new to APBS and I’m very interested in calculating the solvation energy of protein and drug-like ligands with APBS. I have some questions about explicit waters and choice of force field/charge method: 

1.	From my experience with PDB2PQR, explicit waters are usually removed when processing a protein structure before proceeding to the pKa calculation. As in “examples/bem-pKa” (example that calculates the pKa ASP66 in Lysozyme), we don’t have explicit waters in the pqr file. 
If there’re buried, non-interchangeable, structural waters in a protein structure, is it ok to include any explicit waters in the pqr file, as part of the protein structure? What is a good model/charge method for such water molecules? 

2.	In “examples/solv” (example that calculates the solvation energy of methanol and methoxide), it seems like the calculation is done at the united atom level. Is it OPLS-UA? Will it improve the accuracy of this calculation, if explicit waters are involved (like a cluster of methoxide with several explicit waters, instead of a bare methoxide)? Will this kind of calculation be useful for estimation of pKa for small molecules like methanol? 

3.	Can you please advise what is the best force field/charge method for drug-like small molecules, for calculating binding free energies in APBS? 

Any comments/suggestions would be greatly appreciated. Thank you for your time and kind advice in advance! 

Many Thanks,
Amy

**Comment by sobolevnrm** [comment]
Hello --

Thank you for your interest.

1. Buried waters should be fine to include.
2. Many implicit solvent force fields have heavy atom radii that are larger than the length of the bond to the hydrogen atoms.  Because the nonpolar hydrogen atoms have minimal charge, they are often left out of the calculation since they do not contribute significantly to the molecular surface or the charge distribution of the molecule.  I am uncertain of what it would mean to calculate the solvation energy of a small molecule with explicit waters bound to it -- you would likely need to add additional steps in the free energy calculation to account for dissociation of the explicit waters.  Likewise with the pKa calculation, you would need to include steps for the configurational ensemble of the explicit waters and how that ensemble changes during the charging process.
3. AM1-BCC/ZAP9 seems to be a [reliable choice for small molecules](https://pubs.acs.org/doi/10.1021/jm070549%2B).

Thanks again,

Nathan

**Reply by rwxayheee** [reply]
Hi Nathan, 

Thank you so much for your kind response! That is super helpful. 

About adding explicit waters to small molecules/ions: I was thinking if a hybrid explicit/implicit solvation model can improve the accuracy of pKa calculation. Like for methoxide maybe a coordinate water can be added to the charged oxygen. I was wondering if the explicit waters could make up the desolvation penalty or reduce errors in situations with highly charged groups. But what you said makes lots of sense. The thermodynamic cycle will contain more steps, and if more than one waters are involved, it will be necessary to also sample the water configuration. It seems like a very complicated problem to me.. 

Thank you again for your time and kind advice! 

Best regards,
Amy
